### ISSUE TITLE
Plex flow no longer working

### ISSUE THREAD

**Issue by elwin23** [Swiftnesses/com.plex.notify#8]
I have been using a workflow to dim the lights when Plex is started on Chromecast, but the flow is no longer working. (it does work when I click Test).
I confirmed the username and password many times to be sure in the settings.
Any way I can troubleshoot this?
many thanks!

**Comment by Swiftnesses** [comment]
Is this still valid? Apologies, I've not had much time...

**Comment by nerdtechse** [comment]
I have the same problem. And run the latest version that got updated today. The debug in cli show nothing after connected.. Even if i tried to start a couple of movies.
This was what ive got.. 
─────────────── Logging stdout & stderr ───────────────
2018-12-07 22:36:35 [log] [App] com.plex.notify running...
2018-12-07 22:36:35 [log] [App] [PLEX] Server Name: G******S
2018-12-07 22:36:35 [log] [App] [PLEX] Server Version: 1.14.0.5470-9d51fdfaa
2018-12-07 22:36:36 [log] [App] [WEBSOCKET] Connected

**Comment by Swiftnesses** [comment]
This is strange, as of 500 downloads I've not seen this, bar you guys...

Is it your server, on the same network as Homey, what version are you running?

The debug *should* show all player activity...

**Comment by nerdtechse** [comment]
Yes it is..  and its on the same network. Im running like the log says 1.14.0.5470.. (Plex pass member) and the version of homey is 1.5.13.. And your app i downloaded the master version from github. Have tried 3 differnt players now.. no output in log.

**Comment by Swiftnesses** [comment]
I'm at a loss @nerdtechse, perhaps the traffic is being blocked? Are you logged in as the server owner (need to check this!). I'll ask a buddy to see of he can help (this was my first app!).

**Comment by nerdtechse** [comment]
No firewall enabled on the plex server so nothing is blocked there.. And yes im using my admin account on the app. Have also tried to delete the app and reinstall. But allt the same..  Is there anything else that i could debug?

**Comment by nerdtechse** [comment]
And i have now found the problem.. If you set a ip network in plex under "List of IP addresses and networks that are allowed without auth". Then the app wont get a token and therefor not getting any information.
